src/components/Capture/capture.js

```diff
@@ -118,43 +78,57 @@ class Capture extends Component {
       return;
     }
 
-    const payload = this.createPayload(blob, base64)
+    const payload = this.initialiseCapturePayload(blob, base64)
     functionalSwitch(this.props.method, {
       document: () => this.handleDocument(payload),
       face: () => this.handleFace(payload)
     })
   }
 
-  createPayload = (blob, base64) => ({id: randomId(), blob, base64})
+  initialiseCapturePayload = (blob, base64) => ({id: randomId(), blob, base64})
 
-  createJSONPayload = ({id, base64}) => JSON.stringify({id, image: base64})
+  validationServicePayload = ({id, base64}) => JSON.stringify({id, image: base64})
 
-  handleDocument(payload) {
-    const { token, documentType, unprocessedCaptures } = this.props
+  handleAutocapture(payload){
+    const { token, unprocessedCaptures } = this.props
     if (unprocessedCaptures.length === this.maxAutomaticCaptures){
       console.warn('Server response is slow, waiting for responses before uploading more')
       return
     }
-    payload = {...payload, documentType}
-    if (this.props.useWebcam) {
-      postToBackend(this.createJSONPayload(payload), token,
-        (response) => this.onValidationServiceResponse(payload.id, response),
-        this.onValidationServerError
-      )
-      this.setState({documentValidations: false})
+    postToBackend(this.validationServicePayload(payload), token,
+      (response) => this.onValidationServiceResponse(payload, response),
+      this.onValidationServerError
+    )
+  }
+
+  handleCaptureFromUploader(payload) {
+    this.validateAndProceed(payload)
+  }
+
+  createDocumentPayload(payload) {
+    const { documentType } = this.props
```

Comment:
Validations for back side are disable on the Onfido API, so it shouldn't be an issue there. It might be an issue for auto-capture as we are sending the back of a document to the is_document validation service. At the moment, the is_document validation service is letting back of document pass, but maybe we should keep this in mind. I can add a comment, shall I open a Jira ticket too?